provisioning/Samples/service/BulkOperationSample/BulkOperationSample.cs

```diff
@@ -84,16 +84,14 @@ public async Task DeleteIndividualEnrollmentsAsync(List<IndividualEnrollment> in
         public async Task QueryIndividualEnrollmentsAsync()
         {
             Console.WriteLine("\nCreating a query for enrollments...");
-            QuerySpecification querySpecification = new QuerySpecification("SELECT * FROM enrollments");
+            var querySpecification = new QuerySpecification("SELECT * FROM enrollments");
 
-            using (Query query = _provisioningServiceClient.CreateIndividualEnrollmentQuery(querySpecification, QueryPageSize))
+            using Query query = _provisioningServiceClient.CreateIndividualEnrollmentQuery(querySpecification, QueryPageSize);
+            while (query.HasNext())
             {
-                while (query.HasNext())
-                {
-                    Console.WriteLine("\nQuerying the next enrollments...");
-                    QueryResult queryResult = await query.NextAsync().ConfigureAwait(false);
-                    Console.WriteLine(queryResult);
-                }
+                Console.WriteLine("\nQuerying the next enrollments...");
+                QueryResult queryResult = await query.NextAsync().ConfigureAwait(false);
+                Console.WriteLine(queryResult);

```

Comment:
What output does one see here? Not evident to me from reading, and curious if it is helpful.

Reply:
I can see this samples show querying the enrollments and do something with it.
If this is the intention, maybe we comment it so and indicate we are printing out the query result as the example action.